src/components/ConnectButtonAvatar.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { ConnectButtonAvatar } from './ConnectButtonAvatar';
+import { render } from '../utils/test-utils';
+
+describe('ConnectButtonAvatar', () => {
+  describe('when props `isConnected` is false', () => {
+    it('renders `MetaMaskIcon`', () => {
+      const { getByTestId } = render(
```

Comment:
All `get...` functions throw an error if the thing doesn't exist. We should use `query...` instead.

```suggestion
      const { queryByTestId } = render(
```

Reply:
resolved